functions/gyro_straight.py

```diff
@@ -25,8 +46,14 @@ def ExponentialEaseIn(t):
         return 0
     return math.pow(2, 10 * (t - 1))
 
+
+
 ### FUNCTION START
 
+### VERY IMPORTANT
+my_hub = PrimeHub()
```

Comment:
instead of putting this into the function here let's name it prime_hub and put it in the merging code.